tests/e2e/e2e_exec_test.go

```diff
@@ -14,6 +15,98 @@ import (
 	"github.com/ory/dockertest/v3/docker"
 )
 
+const (
+	binaryName = "gaiad"
+
+	flagFrom           = "from"
+	flagHome           = "home"
+	flagFees           = "fees"
+	flagGas            = "gas"
+	flagOutput         = "output"
+	flagChainID        = "chain-id"
+	flagBroadcastMode  = "broadcast-mode"
+	flagKeyringBackend = "keyring-backend"
+)
+
+type flagOption func(map[string]interface{})
+
+// withKeyValue add a new flag to command
+func withKeyValue(key string, value interface{}) flagOption {
+	return func(o map[string]interface{}) {
+		o[key] = value
+	}
+}
+
+func applyOptions(chainID, home string, options []flagOption) map[string]interface{} {
+	opts := map[string]interface{}{
+		flagKeyringBackend: "test",
+		flagOutput:         "json",
+		flagGas:            "auto",
+		flagFrom:           "alice",
```

Comment:
Perhaps we pass `from` so we can reuse this elsewhere

Reply:
if we pass `from` we will overwrite the default from flag. `alice` is only the default one. eg: https://github.com/cosmos/gaia/pull/1738/files#diff-8f39b3a78fd9e2c7c36381bbd6f05c53940c1fb1c7f4babf538ed571cb756b5fR212-R214